packages/docs/content/migration-guides/standardize-theme-colors.mdx

```diff
@@ -7,17 +7,15 @@ intro: We've updated theme colors by removing non-semantic colors red, gold, gre
 import ColorMigrationTable from '../../src/components/content/ColorMigration/ColorMigrationTable.tsx';
 import { Alert } from '@cmsgov/design-system';
 
+## What this changes
+
 With the expansion of the design system themes to include a fourth theme (CMS.gov), we noticed theme variables were inconsistent. In some cases colors and whole color palettes weren't being used and needed to be removed. We've cleaned up and reorganized these colors using design tokens and theming to provide a more intentional approach to color.
 
 ## Brand colors 
 
 ### Primary
 
-We've updated the `primary` color to complete the spectrum from lightest to darkest. 
```

Comment:
Decided to pull this out of the alert styling b/c there isn't any action for someone to take